docs/reference/esql/functions/st_y.asciidoc

```diff
@@ -0,0 +1,33 @@
+[discrete]
+[[esql-st_y]]
+=== `ST_Y`
+
+*Syntax*
+
+[.text-center]
+image::esql/functions/signature/st_y.svg[Embedded,opts=inline]
+
+*Parameters*
+
+`point`::
+Expression of type `geo_point` or `cartesian_point`. If `null`, the function returns `null`.
+
+*Description*
+
+Special function extracts the `x` coordinate from the supplied point.
+If the points is of type `geo_point` this is equivalent to extracting the `longitude` value.
```

Comment:
Same for `longitude`. If I'm not mistaken, it should be latitude.

Reply:
Fixed in 379d6cdd5577a5def64ae4fece1351bac7e64da3